.github/workflows/CI.yaml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ "4.2" ]
+  pull_request:
+    branches: [ "4.2" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        run: ./gradlew build --continue --debug
```

```suggestion
        run: ./gradlew build --continue
```

I think we should remove `--debug` otherwise it generate a lot of noise, in your branch the build is red and I cannot understand why

---

yep, it's temporary, since the build currently does not pass, I wanted to troubleshoot why
sorry, I should have flagged this PR as draft